mtags/src/main/scala/scala/meta/internal/mtags/WalkFiles.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.internal.mtags
+
+import java.nio.file.FileVisitResult
+import java.nio.file.Files
+import java.nio.file.NoSuchFileException
+import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.attribute.BasicFileAttributes
+import scala.meta.io.AbsolutePath
+import scala.collection.mutable.ArrayBuffer
+
+object WalkFiles {
```

Comment:
Maybe add to extension method? It's really hidden in here, if someone doesn't know about this then they will use something else.